java/kotlin-extractor/src/main/kotlin/KotlinFileExtractor.kt

```diff
@@ -595,6 +595,8 @@ open class KotlinFileExtractor(
                 tw.writeMethods(clinitId, "<clinit>", "<clinit>()", returnType.javaResult.id, parentId, clinitId)
                 tw.writeMethodsKotlinType(clinitId, returnType.kotlinResult.id)
 
+                tw.writeCompiler_generated(clinitId, 4)
```

Comment:
Define constants / an enum for these integers?

Reply:
Done. I've put it in `KotlinFileExtractor` for now, but we might want to move it out later if we have generated things elsewhere.